admin_manual/configuration/user/user_types.rst

```diff
@@ -0,0 +1,82 @@
+===================
+ownCloud User Types
```

Comment:
Please call it "ownCloud roles" (or "ownCloud role types" ?). Users (meaning technically in the software) are just a part of that.